modules/operations/pages/astream-pricing.adoc

```diff
@@ -1,5 +1,72 @@
-= {product_name} Pricing
+= {product} pricing
 :page-tag: astra-streaming,planner,plan,pulsar
 
-For information on *pricing* for {product_name}, see https://www.datastax.com/products/astra-streaming/pricing[Astra Streaming pricing].
+{product} is a fully-managed, software-as-a-service (SaaS) offering embedded in {astra_db}.
 
+{product} offers three subscription plans, as well as pricing for dedicated clusters.
+A {product} subscription is associated with an {astra_db} organization, but {product} subscription plans are separate from {astra_db} organization subscription plans.
```

Comment:
This is nit-picking :)

Should be: An {product} subscription ...

Reply:
It's not nitpicking because the result is grammatically incorrect. 😄  I mess this up all the time because my brain goes "A product subscription", cause that's what I'm typing, but that's not correct 🤪